src/bentoml/_internal/cloud/deployment.py

```diff
@@ -300,16 +295,12 @@ def get_bento_info(
     if project_path:
         from bentoml.bentos import build_bentofile
 
-        with Live(_cloud_client.spinner.progress_group, transient=True):
-            with _cloud_client.spinner.spin(
-                text=f"🍱 Building bento from project: {project_path}"
-            ):
+        with _cloud_client.spinner as spinner:
```

Comment:
Will this produce the desired effect without adding `transient=True`?

<img width="499" alt="image" src="https://github.com/bentoml/BentoML/assets/1206493/bd3e1f4c-8175-4ca5-9e53-6ed399ca6591">
<img width="876" alt="image" src="https://github.com/bentoml/BentoML/assets/1206493/85eb25cd-b4dc-4ea4-abc0-b0f58f80df9d">

Reply:
I don't think so. But we don't need to clear the render either because `Built Bento` and `Pushed Bento` lines will be displayed in the `spinner` now.